zendframework/zend-validator/ZF2015-03.yaml

```diff
@@ -0,0 +1,17 @@
+title:     Invalid CSRF validation of null or incorrectly formatted token identifiers
+link:      http://framework.zend.com/security/advisory/ZF2015-03
+cve:       CVE-2015-1786
+branches:
+    2.0.x:
+        time:     2015-03-12 13:58:47
+        versions: [>=2.0.0,<2.0.99]
```

Comment:
Is this right? The blog post states:

> Note: in testing the patch, we discovered that the vulnerability was introduced specifically in the 2.3 series, and thus no patch was necessary against the 2.2 series.

So I would assume that you can remove the entries for `2.0.x`, `2.1.x` and `2.2.x`.